.gitignore

```diff
@@ -12,4 +12,4 @@ settings.json
 out/
 dist/
 .vscode-test
-package-lock.json
```

Comment:
This new feature of npm guarantees that each developer builds the project with the same package versions. So it is a best practice to push this file in our git repository.
[https://stackoverflow.com/questions/44297803/i-am-confused-package-lock-json-role](https://stackoverflow.com/questions/44297803/i-am-confused-package-lock-json-role)